cfme/physical/provider/__init__.py

```diff
@@ -14,14 +14,14 @@
 
 class PhysicalProvider(Pretty, BaseProvider, Fillable):
     """
-    Abstract model of an infrastructure provider in cfme. See VMwareProvider or RHEVMProvider.
+    Abstract model of a physical infrastructure provider in cfme. See LenovoProvider.
     """
     provider_types = {}
     category = "physical"
     pretty_attrs = ['name']
-    STATS_TO_MATCH = ['num_server']
-    # string_name = "Physical Infrastructure"
-    # page_name = "infrastructure"
+    STATS_TO_MATCH = ['num_server', 'num_server_with_host']
+    string_name = "Physical Infrastructure"
```

Comment:
**ANNOTATION:** This is the string that is used when the flash messages say things like, ``Add of Physical Infrastructure Provider was cancelled by the user```